aten/src/ATen/native/cpu/DistributionTemplates.h

```diff
@@ -226,12 +229,15 @@ struct UniformKernel {
 // ==================================================== Cauchy ========================================================
 
 template<typename RNG>
-void cauchy_kernel(TensorIterator& iter, double median, double sigma, RNG generator) {
+void cauchy_kernel(TensorIterator& iter, double median_, double sigma_, RNG generator) {
   AT_DISPATCH_FLOATING_TYPES_AND_HALF(iter.dtype(), "cauchy_cpu", [&]() {
+    using accscalar_t = at::acc_type<scalar_t, false>;
```

Comment:
Reduction of precision of median/sigma when you half/bfloat16. This change is probably benign but it is technically BC-breaking and not remarked upon in the notes. Also, I noticed in another part of the PR in `normal` you retained double-precision distribution for compatibility (`at::normal_distribution<double> normal(mean, std); // `double` instead of `scalar_t` for compatibility`) which is inconsistent with the change here.